molecule/provisioner/ansible.py

```diff
@@ -202,11 +202,11 @@ class Ansible(base.Base):
     ::
 
         ANSIBLE_ROLES_PATH:
-          $ephemeral_directory/roles/:$project_directory/../
```

Comment:
The library -> modules and filters -> filter path changes are breaking changes or?

Reply:
Yes but only if a user has overridden these variables in the molecule.yml. Otherwise the new paths are correctly expanded by the code change in ansible.py

Reply:
But it shouldn't be that much of an issue since the documentation was already wrong. The path was `$ephemeral_directory/libraries` in the code but `$ephemeral_directory/library` in the docs. If have not seen any issues about it. Also the library only contains the internal vagrant module.

The filters plugin could break by the point above.

Reply:
Uhhhh, right. Well, I'm not sure what to do then. We should perhaps include a more clarifying note in the change log that this could break things for end-users? @ssbarnea what do you think here?